src/analysis/max_min_driving_force.jl

```diff
@@ -0,0 +1,150 @@
+"""
+    max_min_driving_force(
+        model::StandardModel,
+        optimizer,
+        thermodynamic_data;
+        modifications=[],
+        proton_id="h_c",
+        water_id="h2o_c",
+        concentration_ratios=[("atp_c", "adp_c", 10.0),
+        ("adp_c", "amp_c", 1.0),
+        ("nadph_c", "nadp_c", 10.0),
+        ("nadh_c", "nad_c", 0.1)],
+        constant_concentrations=[("coa_c", 1e-3),
+        ("co2_c", 10e-6),
+        ("pi_c", 10e-3),
+        ("ppi_c", 1e-3)],
+        concentration_lb=1e-6,
+        concentration_ub=10e-3)
+
+Perform max min driving force analysis on `model` using `optimizer` and
```

Comment:
what does it actually do?

```suggestion

Using the gibbs free energies in `gibbs_energies`, find the minimum and maximum flux (?) of the reactions.
```

The return type/structure must be documented. I'd suggest a named tuple.

Reply:
Yes, good idea with the named tuple. I was returning 3 `nothings` since the function used to return 3 variables and I didn't want an error because of too many arguments on the user side if the function fails...

What does it actually do, ha ha it finds the maximum minimum driving force. It is a mouthful but the name is actually exactly what it does :)